### PR TITLE
Update ch09

### PR DESCRIPTION
**Pull request by jrycw**
Some tweaks for ch09

### CODE REVIEW

File: src/ch09-02-recoverable-errors-with-result.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ copy and paste relevant text
 -->
 
 ```text
-thread 'main' panicked at 'hello.txt should be included in this project: Os {
+thread 'main' panicked at 'hello.txt 應該要存在此專案中: Os {
```
Comment (weihanglo) [inline src/ch09-02-recoverable-errors-with-result.md:139]: Why changed this?

IIRC l10n in rust compiler diagnostics is not yet rolled out

**Review by weihanglo — COMMENTED** [review]
Others LGTM. Thanks!

File: src/ch09-02-recoverable-errors-with-result.md (hunk 1 above)
Comment (jrycw) [reply src/ch09-02-recoverable-errors-with-result.md:139]: ok. I originally thought the message should be matched with the [code](https://github.com/rust-tw/book-tw/blob/zh_tw/listings/ch09-error-handling/no-listing-05-expect/src/main.rs).

Comment (weihanglo) [reply src/ch09-02-recoverable-errors-with-result.md:139]: My fault. You're absolutely right. I thought it was a message from rustc.

**Review by weihanglo — APPROVED** [review]

**Comment by weihanglo** [comment]
Thanks!

**Pull request merged**